src/TaskfileInstallerPlugin.php

```diff
@@ -142,4 +144,18 @@ private function installGitignore(): void
             }
         }
     }
+
+    /**
+     *
+     */
+    private function installDdevCommand(): void
+    {
+        if (file_exists('./.ddev/config.yaml')) {
```

Comment:
I wonder if we should document this somewhere. I imagine an existing project pulling in drainpipe first, then later pulling in ddev, and not knowing that running `composer update lullabot/drainpipe` would be a benefit. Perhaps in the readme for now?